### PR TITLE
Add datasheet to /cloud/managed-cloud

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* Add datasheet to /cloud/managed-cloud
* Added many Mark Baker text changes that I found in the document
* Fixed or removed EMPTY HREFs!!!
* Used {{ ASSET_SERVER_URL }} instead of actual link

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/cloud/managed-cloud)
- compare to the [copy doc](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit#)

### CODE REVIEW

File: templates/cloud/managed-cloud.html [hunk 1]
```diff
@@ -326,7 +330,7 @@ <h3 class="p-list-step__title">
                 <span class="p-list-step__bullet">1</span>
                 We <strong>build</strong> your cloud
               </h3>
-              <p>Together, we <a href="">design your OpenStack cloud</a> based on your hardware, scale, roadmap, applications and monitoring system. This happens on your site, in your datacentre or hosted at a third-party facility.</p>
+              <p>Together, we <a href="cloud/foundation-cloud">design your OpenStack cloud</a> based on your hardware, scale, roadmap, applications and monitoring system. This happens on your site, in your datacentre or hosted at a third-party facility.</p>
```
Comment (anthonydillon) [inline templates/cloud/managed-cloud.html]: This should be `/cloud/foundation-cloud`

**Comment by anthonydillon** [comment]
## QA issues:
### Hero
- The title in the copy doc is `Fully managed Cloud, by Canonical`
- The intro text is different from the copy doc

### Canonical secures the OpenStack majority
- The title in the copy doc is "Security trusted by the OpenStack majority"
- The first paragraph is different from the doc
- The cite of the pie chart diagram is "Ubuntu powers 55% of OpenStack in production Source: OpenStack Foundation Survey April 2017" in the copy doc

### Fits your hybrid cloud strategy
- The sentence "The right location for a workload should be driven by economics, not habit." is not in the copy doc
- Managed OpenStack is "Fully Managed OpenStack" in the copy doc

### BOOTSTACK HELPS THESE COMPANIES
- The title in the copy doc is "BootStack fully managed cloud customers"
- Missing Cogeco Datacenters, Fibernet logos

### Get BootStack — OpenStack managed cloud in 3 steps
- The button should link to `/cloud/contact-us?product=managed-cloud`

### Shape it to your needs
- There is no link "Read the QTS press release" in the copy doc
- OpenStack build service › link should not be external
- The link "Use Linux Containers Project, LXD" should go to linuxcontainers.org
- professional services and consulting link should go to `/openstack/consulting`
- Let’s discuss your requirements button should be to `/cloud/contact-us?product=managed-cloud`

**Comment by pmahnke** [comment]
@anthonydillon thanks for the great read through... I think these were mostly not highlighted... my fault though.

- 'The sentence "The right location for a workload should be driven by economics, not habit." is not in the copy doc' it is there.
- The link - The button should link to /cloud/contact-us?product=managed-cloud - is wrong, I fixed the copy doc
- This link 'professional services and consulting link should go to /openstack/consulting' doesn't exist... sent it to foundation cloud for now and put a comment in the copy doc
- This link 'Let’s discuss your requirements button should be to /cloud/contact-us?product=managed-cloud' is wrong

**Comment by frankiegranato** [comment]
Two redirects need to be added. See copydoc.

**Comment by anthonydillon** [comment]
## QA issues:

### Fits your hybrid cloud strategy
- There is still a duplicate line: "The right location for a workload should be driven by economics, not habit."

**Comment by anthonydillon** [comment]
@pmahnke One tiny thing left.

**Comment by pmahnke** [comment]
@frankiegranato 

I am really confused by all the redirects between the copy doc and [spreadsheet](https://docs.google.com/spreadsheets/d/1FyLhb0aa7Uo6D4jzBDOHrCO2geZ1twAtCjD06YdEK6A/edit)

I think I should move it to /openstack/managed and then redirect things to there?

**Comment by pmahnke** [comment]
@frankiegranato I am going to push this as there are LOT OF ERRORS on the page.

when you get a sec, we can talk about redirects and urls

**Pull request merged**